client/components/basic/RoomInfo.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Box, Margins, Icon, Button, ButtonGroup, Divider } from '@rocket.chat/fuselage';
+import { css } from '@rocket.chat/css-in-js';
+
+import RoomAvatar from './avatar/RoomAvatar';
+import { useTranslation } from '../../contexts/TranslationContext';
+import UserCard from './UserCard';
+import VerticalBar from './VerticalBar';
+
+const wordBreak = css`
+	word-break: break-word;
+`;
+
+const Label = (props) => <Box fontScale='p2' color='default' {...props} />;
+const Info = ({ className, ...props }) => <UserCard.Info className={[className, wordBreak]} flexShrink={0} {...props}/>;
+
+const RoomInfoFooter = () => (
+	<Box p='x24'>
+		<Margins block='x12'>
+			<ButtonGroup stretch>
+				<Button><Box is='span' mie='x4'><Icon name='eye-off' size='x20' /></Box>Hide</Button>
+				<Button danger><Box is='span' mie='x4'><Icon name='sign-out' size='x20' /></Box> Leave</Button>
+			</ButtonGroup>
+
+			<Divider />
+
+			<ButtonGroup stretch>
+				<Button><Box is='span' mie='x4'><Icon name='edit' size='x20' /></Box>Edit</Button>
+				<Button danger><Box is='span' mie='x4'><Icon name='trash' size='x20' /></Box> Trash</Button>
+			</ButtonGroup>
+		</Margins>
+	</Box>
+);
+
+export const RoomTitle = ({ roomName }, props) => <Box mbs='x16' fontScale='s2' color='default' {...props}><Icon name='lock' />{roomName}</Box>;
```

Comment:
I think it should be part of a possible new component, like 'RoomCard' -> RoomCard.Title